src/components/BarChart.js

```diff
@@ -10,42 +10,45 @@ const BarChart = ({
   size = { width: 600, height: 400 },
   margin = { top: 20, right: 20, bottom: 30, left: 50 },
   labels = true,
+  fill = 'firebrick',
   data,
 }) => {
   const height = size.height - margin.top - margin.bottom
   const parse = timeParse('%Y-%m-%d')
   const dataClean = data.map(d => ({ date: parse(d[0]), value: +d[1] }))
```

Comment:
i think the bar chart should not assume that the x variable is a date (as opposed to the timechart) -- we probably want it be able to handle data that looks like `[['foo', 4], ['bar', 10]]`

i would nix `L17` and update 18 to something like `const dataClean = data.map(d => ({ x: d[0], y: +d[1] }))`